federated-learning-controller/docs/Install-OTel-Addon.md

```diff
@@ -0,0 +1,174 @@
+# Multi-Cluster Kubernetes Sustainability Monitoring with OpenTelemetry
+
+This project provides a comprehensive solution for monitoring sustainability and performance metrics across a multi-cluster Kubernetes environment managed by [Open Cluster Management (OCM)](https://open-cluster-management.io/).
+
+It leverages [Kepler](https://github.com/sustainable-computing-io/kepler) to export energy consumption metrics, an [OpenTelemetry (OTel) Collector](https://opentelemetry.io/docs/collector/) to aggregate and forward these metrics, and [Prometheus](https://prometheus.io/) for centralized storage and querying.
+
+## Architecture
+
+The monitoring system is designed with a hub-and-spoke architecture:
+
+*   **Hub Cluster**: A central OCM management cluster that runs Prometheus to aggregate and store metrics from all managed clusters.
+*   **Spoke Clusters (cluster1, cluster2)**: Managed clusters that run Kepler and the OpenTelemetry agent to collect local sustainability metrics and forward them to the Hub Cluster.
+
+![Architecture Diagram](../assets/images/obs_architecture.png)
+
+## Prerequisites
+
+Before you begin, ensure you have the following tools installed:
+
+*   [Kind (Kubernetes in Docker)](https://kind.sigs.k8s.io/)
+*   `kubectl`
+*   `git`
+*   `helm`
+
+## Test Environment Setup
+
+Please verify that the clusters are registered with the hub:
+
+```bash
+kubectl --context kind-hub get managedclusters
+```
+
+You should see an output similar to this:
+```text
+NAME            HUB ACCEPTED   MANAGED CLUSTER URLS                  JOINED   AVAILABLE   AGE
+cluster1        true           https://cluster1-control-plane:6443   True     True        3h36m
+cluster2        true           https://cluster2-control-plane:6443   True     True        3h36m
+local-cluster   true           https://hub-control-plane:6443        True     True        3h35m
+```
+
+---
+
+## Deployment Guide
+
+Follow these steps to deploy the monitoring stack and the OpenTelemetry add-on.
```

Comment:
summary: enable obs -> prom collect ...   -> otel addon collect/report metric to hub prom server...